css/main.sass

```diff
@@ -453,15 +458,20 @@ footer ul
             overflow: hidden
         .social
             align-items: flex-start
-            display: inherit
+            display: inline
             flex-direction: row
-            order: 2
-            margin: 0px
             div:not(:first-of-type)
-                margin-left: 10px
+                margin-left: 20px
             img
                 height: 45px
                 width: 45px
+        .social-container
```

Comment:
Here as well—avoid styling the same thing in multiple places

Reply:
How would I go about changing styling depending on screen size if I don't reference the item in separate locations?

Reply:
You're right, I didn't expand the file so I didn't see these were under separate @media queries—my bad!